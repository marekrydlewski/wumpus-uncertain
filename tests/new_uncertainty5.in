5 4
0.7
??B?
B???
O??O
B??B
????
