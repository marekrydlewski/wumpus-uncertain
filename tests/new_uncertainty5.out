0.77 0.72 0.00 0.72 
0.00 0.77 0.72 0.00 
0.00 0.00 0.00 0.00 
0.00 0.77 0.77 0.00 
0.77 0.70 0.70 0.77 
